src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -368,6 +368,23 @@ public override TypeWithAnnotations ReturnTypeWithAnnotations
             }
         }
 
+        public override FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations
+        {
+            get
+            {
+                var result = FlowAnalysisAnnotations.None;
+                if (_property.MaybeNullAttributeIfExists is object)
```

Comment:
>if (_property.MaybeNullAttributeIfExists is object) [](start = 16, length = 51)

Implementation of this property should rely on well known attributes infrastructure. Please restore that.